File: 3st/Seojun/각도기.java
class Solution {
    public int solution(int angle) {
        int as = 0;
        if(angle>90&&angle<180){
            as = 3;
        }
        else if(angle<90&&angle>0){
            as = 1;
        }
        else if(angle==90){
            as =2;
        }
        else if(angle == 180){
            as = 4;
        }
        int answer = as;
        return answer;
    }
}

File: 프로그래머스/0/120829. 각도기/각도기.py
def solution(angle):
    answer = 0
    if angle < 90 :
        answer = 1
    elif angle == 90 :
        answer = 2
    elif angle > 90 and angle < 180 :
        answer = 3
    elif angle == 180 :
        answer = 4
    return answer